static/js/lib/ecommerce_test.js

```diff
@@ -40,6 +42,82 @@ describe("ecommerce", () => {
     })
   })
 
+  describe("calculateTax", () => {
+    it("calculates the tax of an item", () => {
```

Comment:
nit: There are 3 similar tests below. Could you convert those into a single parameterized test? It should take price, coupon value, tax, and expected amount in the params.

Reply:
Should be resolved in https://github.com/mitodl/mitxpro/pull/2779/commits/b3070a71d3d4fa1cc88cfffbd1bff09bf17a35b6